test/helpers/envtest.go

```diff
@@ -118,10 +203,115 @@ func NewTestEnvironment() *TestEnvironment {
 	}
 }
 
+const (
+	mutatingWebhookKind   = "MutatingWebhookConfiguration"
+	validatingWebhookKind = "ValidatingWebhookConfiguration"
+	mutatingwebhook       = "mutating-webhook-configuration"
+	validatingwebhook     = "validating-webhook-configuration"
+)
+
+func appendWebhookConfiguration(mutatingWebhooks []runtime.Object, validatingWebhooks []runtime.Object, configyamlFile []byte, tag string) ([]runtime.Object, []runtime.Object, error) {
+
+	objs, err := utilyaml.ToUnstructured(configyamlFile)
+	if err != nil {
+		klog.Fatalf("failed to parse yaml")
+	}
+	// look for resources of kind MutatingWebhookConfiguration
+	for i := range objs {
+		o := objs[i]
+		if o.GetKind() == mutatingWebhookKind {
+			// update the name in metadata
+			if o.GetName() == mutatingwebhook {
+				o.SetName(strings.Join([]string{mutatingwebhook, "-", tag}, ""))
+				mutatingWebhooks = append(mutatingWebhooks, &o)
+			}
+		}
+		if o.GetKind() == validatingWebhookKind {
+			// update the name in metadata
+			if o.GetName() == validatingwebhook {
+				o.SetName(strings.Join([]string{validatingwebhook, "-", tag}, ""))
+				validatingWebhooks = append(validatingWebhooks, &o)
+			}
+		}
+	}
+	return mutatingWebhooks, validatingWebhooks, err
+}
+
+func initializeWebhookInEnvironment() {
```

Comment:
Might add a comment explaining why we are mutating webhooks definitions instead of using existing manifests (to avoid name clash)